request.js

```diff
@@ -1413,6 +1413,28 @@ Request.prototype.jar = function (jar) {
   return self
 }
 
+Request.prototype.cloneJar = function (cookieJar) {
+  var self = this
+  var newJar, cookieString, cookieStore, domains, domainPaths, cookieUrl
+
+  newJar = self.jar()
+  cookieStore = cookieJar._jar.store.idx
```

Comment:
Seems like we want a supported way to get all the cookies from the cookie store without reaching into "private" interfaces. We may want to discuss this with tough-cookie directly.